ios/FluentUI.Tests/DatePickerControllerTests.swift

```diff
@@ -92,7 +92,16 @@ class DatePickerControllerTests: XCTestCase {
 
 class MockCalendarViewStyleDataSource: CalendarViewStyleDataSource {
     func calendarViewDataSource(_ dataSource: CalendarViewDataSource, textStyleForDayWithStart dayStartDate: Date, end: Date, dayStartComponents: DateComponents, todayComponents: DateComponents) -> CalendarViewDayCellTextStyle {
-        if dayStartComponents.dateIsTodayOrLater(todayDateComponents: todayComponents) {
+        if dayStartComponents.dateIsInCurrentMonth(todayDateComponents: todayComponents) {
```

Comment:
Nit: maybe a ternary statement? something like:
`return dayStartComponents.dateIsInCurrentMonth(todayDateComponents: todayComponents) ? .primary : .secondary`

Reply:
And with other areas of code